test/arrays/diten/bigArrayAsVec.chpl

```diff
@@ -0,0 +1,8 @@
+config const n = 1024*1024;
+
+on Locales[numLocales-1] {
+  var arr : [0..-1] int;
+  for i in 1..n {
+    arr.push_back(i);
+  }
+}
```

Comment:
I don't think this test will actually trigger the original bug. You need an array that's at least 2 hugepages to trigger the code path that was failing. You also don't need the on-stmt, you just need to run it on at least 2 locales so hugepages are used.

I think this could just be:

```chpl
config const n = 8*1024*1024;

var arr : [0..-1] int;
for i in 1..n do 
  arr.push_back(i);
```

And if we want it to run in a ugni config, it needs to be in test/runtime/configMatters

Reply:
Thanks! I changed the test as you suggested and moved it to test/runtime/configMatters/comm/ugni.  Look good now?

Reply:
The ugni subdir is only run for ugni, and I think this is worth testing in other configurations beyond just ugni in case we run into similar problems in other configs. Maybe just pull it up one level and leave it in the comm dir?